Added project and About section to index file

shurichmond opened the pull request:
Finished  Mobile version

- [x] Added projects and About sections to index
- [x] Checked all Linters Errors
- [x] Used required README

shurichmond committed:
Added project and About section to index file

Nyame-Wolf commented on index.html:
```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
         </ul>
     </section>
+
+    <section class="Projects">
```
I suggest you ensure this section follows [project instructions]() and uses `display:grid`. The section should have use `display:grid` and you can let the individual cards/articles remain with `display:flex`
The screenshot below illustrates this instruction:
<img width="1666" alt="Screenshot 2023-01-13 at 18 30 32" src="https://user-images.githubusercontent.com/48546992/212357781-91343f6e-34e5-49de-aa33-2b5f24e6715d.png">

Nyame-Wolf commented on index.html:
```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
+                    <span class="year">2015</span>
+    
+                </ul>
+                <span class="details">
```
I suggest you use a `<p>` tag for this paragraph here. Using the right html element ensures screen readers are not confused when they parse your site.

Nyame-Wolf commented on index.html:
```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
+        </section>
+
+        <section class="Card-Works">
+            <header class="Snapshot">
```
I suggest you remove this header element here. Header element are mostly used for navigation and your page should follow a `header -> main -> footer` theme. Having this header element at the center of your app confuses screen readers. To learn more [read](https://www.w3schools.com/tags/tag_header.asp)

Nyame-Wolf commented on index.html:
```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
+
+        <section class="Card-Works">
+            <header class="Snapshot">
+                <img src="./Img/Snapshoot Portfolio (1).png" alt="snapshot poortfolio" class="snapshot">
```
This image link is not visible. Its link is broken. I would suggest that instead of using an `img` tag here, look into making this a [background image](https://www.w3schools.com/cssref/pr_background-image.php). 
- Decorations are always backgrounded images, not `img` tags)

Nyame-Wolf commented on index.html:
```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
+                <img src="./Img/Snapshoot Portfolio (1).png" alt="snapshot poortfolio" class="snapshot">
+
+            </header>
+            <div class="info-block">
```
I suggest that you kindly ensure that the cards use the same margins and paddings as the figma design. This is part of the project requirements
<img width="1449" alt="Screenshot 2023-01-13 at 18 49 08" src="https://user-images.githubusercontent.com/48546992/212361532-acfdce30-6c8a-4581-b5f4-934ace1b64c1.png">

This is your design

<img width="678" alt="Screenshot 2023-01-13 at 18 44 13" src="https://user-images.githubusercontent.com/48546992/212361247-0df427f9-d1fd-43b9-9ff1-ce1c3018c689.png">

this is the figma template
<img width="378" alt="Screenshot 2023-01-13 at 18 43 26" src="https://user-images.githubusercontent.com/48546992/212361307-02769c57-5b8d-4351-9544-c7527aa693b2.png">

Nyame-Wolf reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle: 

Hi @shurichmond  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 

![](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

## Highlights :sparkles:

- Great work using the right GitHub Flow :100: : 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

shurichmond committed:
Modified index and css files

shurichmond commented:
Completed Required changes

- [x]  display: grid Used
- [x]  <p> tag used where suggested
- [x] <header> element changed where suggested
- [x] modified img path, so it displays properly now
- [x] Modified Margins as Figma template

Owusu-Desmond reviewed (approved):
Hi @shurichmond , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media3.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
